.circleci/config.yml

```diff
@@ -0,0 +1,42 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/pkcs11
+    docker:
+      - image: circleci/ubuntu-server:latest
```

Comment:
This is the reason why Circle-CI was used, quick and easy way to test on the latest Ubuntu images.